tests/groupby_test.py

```diff
@@ -202,104 +202,135 @@ def test_boolean_arrays(self):
 
     def test_bitwise_aggregations(self):
         revs = ak.arange(self.igb.length) % 2
-        self.assertTrue((self.igb.OR(revs)[1] == self.igb.max(revs)[1]).all())
-        self.assertTrue((self.igb.AND(revs)[1] == self.igb.min(revs)[1]).all())
-        self.assertTrue((self.igb.XOR(revs)[1] == (self.igb.sum(revs)[1] % 2)).all())
+        self.assertListEqual(
+            self.igb.OR(revs)[1].to_ndarray().tolist(), self.igb.max(revs)[1].to_ndarray().tolist()
+        )
+        self.assertListEqual(
+            self.igb.AND(revs)[1].to_ndarray().tolist(), self.igb.min(revs)[1].to_ndarray().tolist()
+        )
+        self.assertListEqual(
+            self.igb.XOR(revs)[1].to_ndarray().tolist(),
+            (self.igb.sum(revs)[1] % 2).to_ndarray().tolist(),
+        )
 
     def test_standalone_broadcast(self):
         segs = ak.arange(10) ** 2
         vals = ak.arange(10)
         size = 100
         check = ((2 * vals + 1) * vals).sum()
-        self.assertTrue(ak.broadcast(segs, vals, size=size).sum() == check)
+        self.assertEqual(ak.broadcast(segs, vals, size=size).sum(), check)
         perm = ak.arange(99, -1, -1)
         bcast = ak.broadcast(segs, vals, permutation=perm)
-        self.assertTrue(bcast.sum() == check)
+        self.assertEqual(bcast.sum(), check)
         self.assertTrue((bcast[:-1] >= bcast[1:]).all())
 
     def test_broadcast_ints(self):
         keys, counts = self.igb.count()
 
-        self.assertTrue((np.array([1, 4, 2, 1, 2]) == counts.to_ndarray()).all())
-        self.assertTrue((np.array([1, 2, 3, 4, 5]) == keys.to_ndarray()).all())
+        self.assertListEqual(np.array([1, 4, 2, 1, 2]).tolist(), counts.to_ndarray().tolist())
```

Comment:
There are a lot in this file that can be defined as a list instead of using `.tolist()`